compiler/tests-spec/testData/diagnostics/linked/overload-resolution/building-the-overload-candidate-set-ocs/operator-call/p-4/neg/1.1.fir.kt

```diff
@@ -17,7 +17,7 @@ fun case1() {
 }
 
 class B() {
-    val p: String by <!TOO_MANY_ARGUMENTS!>Delegate()<!> // DELEGATE_SPECIAL_FUNCTION_MISSING expected
+    val p: String by <!DELEGATE_SPECIAL_FUNCTION_NONE_APPLICABLE!>Delegate()<!> // DELEGATE_SPECIAL_FUNCTION_MISSING expected
```

Comment:
`// DELEGATE_SPECIAL_FUNCTION_MISSING expected` ?

Reply:
Thanks! Actually I think in this case it should be `DELEGATE_SPECIAL_FUNCTION_NONE_APPLICABLE` instead because the candidate is indeed available (`getValue` property in `Delegate`). But the type of `G.invoke` does not match the expected type.

As for now, FE1.0 crashes when compiling this code. I believe https://youtrack.jetbrains.com/issue/KT-3568 is tracking it. The FIR backend is able to correrctly compile this code after I fix the signature of `G.invoke` and the resulted program can run correctly (Yay!!!).